agent/bench-scripts/driver/linpack

```diff
@@ -225,16 +231,22 @@ while true; do
 	esac
 done
 
-binary=${1}
-if [[ -z "${binary}" ]]; then
-	printf -- "[%s] ERROR: You must specify the location of the LINPACK binary\n\n" "${script_name}" >&2
+ver=${1}
+if [[ -z "${ver}" ]]; then
+	printf -- "[%s] ERROR: You must specify the version of the LINPACK binary\n\n" "${script_name}" >&2
 	usage >&2
 	exit 2
 fi
+binary="/usr/local/pbench-linpack-${ver}/benchmarks/linpack/xlinpack_xeon64"
```

Comment:
This is highly opinionated!  How does this file get created, or, how does the user know to put this file here?

Reply:
Your subsequent change improved this.